go.mod

```diff
@@ -38,5 +38,5 @@ require (
 	github.com/jrick/logrotate v1.0.0
 	github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a
-	golang.org/x/sys v0.0.0-20211216021012-1d35b9e2eb4e
+	golang.org/x/sys v0.5.0
```

Comment:
Intentionally not updating to latest https://github.com/golang/sys/releases/tag/v0.7.0?

Reply:
Oh same reason as dcrwallet presumably

Reply:
Right.  Also, because master is still on 0.5.0 too and I don't want to get the release branch ahead.